build-npm/action.yml

```diff
@@ -0,0 +1,87 @@
+name: Build NPM
+descripion: Build a npm-module
+
+inputs:
+  github_artifact_retention:
+    description: 'Time during which artifact are kept in Github infrastructure'
+    default: '2'
+  tests_module_path:
+    description: 'Path to a folder in the repository containing a tests module to be built'
+    required: false
+    default: 'tests/jahia-module/'   
+  node_version:
+    description: 'Version of node to install on the host'
+    required: false
+    default: 'lts/*'
+
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/setup-node@v3
+      with:
+        node-version: ${{ inputs.node_version }}
+
+    - name: Install jq
+      shell: bash
+      run: sudo apt-get install jq
+
+    - name: Get package.json infos
+      shell: bash
+      run: |
```

Comment:
I feel it would be nice to have 2-3 comments here to explain why you're doing what you're doing here (the whole thing about aligning to the "maven" way of doing things.